src/java.base/share/classes/sun/nio/ch/UnixDomainSockets.java

```diff
@@ -178,4 +182,8 @@ private static native int accept0(FileDescriptor fd, FileDescriptor newfd, Objec
         IOUtil.load();
         supported = init();
     }
+
+    static UnixDomainSocketAddress getUNNAMED() {
```

Comment:
Can you rename this to unnamed() and move it up with the other package default static methods?

Reply:
Thanks for reviewing!
Renamed to `unnamed()`.